src/Display/DisplatTft.h

```diff
@@ -0,0 +1,20 @@
+#pragma 0
```

Comment:
Shouldn't this be `once` instead of `0`?
```suggestion
#pragma once
```